libraries/config/arch/arm64/cpu/cortex_a57/cpu.mk

```diff
@@ -1,5 +1,5 @@
 export ATMOSPHERE_DEFINES  += -DATMOSPHERE_CPU_ARM_CORTEX_A57
-export ATMOSPHERE_SETTINGS += -mtune=cortex-a57
+export ATMOSPHERE_SETTINGS += -march=armv8-a+crc+crypto -mtune=cortex-a57
```

Comment:
Should this be a board thing? I guess this is the right place to put it...

Reply:
Actually I'm not sure. Arm designed processors can be configured with or without the extensions by system implementors. I guess technically it should be done on a board by board basis; but since the only A57 board we target does have the crypto extensions, for now this will do.